devtools/conda-envs/alchemiscale-client.yml

```diff
@@ -24,12 +24,12 @@ dependencies:
   - nest-asyncio
 
   # openmm protocols
```

Comment:
Apologies in advance, I suspect I'm being silly but just wanted to ask "in case" - should pontibus be added to the pip install lists in this PR too?

Reply:
No, not here. We will do that in the `alchemiscale.org-deployment` repo for the `root` deployment!